### ISSUE TITLE
Refactor: [Qwen3] Rename `K_MASK` to `_K_MASK` for convention consistency

### ISSUE THREAD

**Issue by gear** [jax-ml/bonsai#13]
The current key mask constant name might cause accidental name collision when import.

```python
# https://github.com/jax-ml/bonsai/blob/main/bonsai/models/qwen3/modeling.py
# Line 27

K_MASK = -2.3819763e38
```

We should change it to:

```python
_K_MASK = -2.3819763e38
```

Optional:
This magic number can be accessed via `jax._src.nn.functions._get_large_negative(jnp.float32).item()`, it might be clearer to assign it as:

```python
# Suppress softmax with large negative (-2.3819763e38)
_K_MASK = jax._src.nn.functions._get_large_negative(jax.numpy.float32).item()
```

**Comment by jenriver** [comment]
Hello gear, thanks for bringing this great point!
We welcome contributions, feel free to shoot a PR to fix this :)

**Comment by gear** [comment]
Hey @jenriver, would it be better to have a separate test case for this variable in `test/`?

**Comment by jenriver** [comment]
Hey @gear, we'd benefit with extensive test cases for sure, especially regarding golden logits check, mask correctness, etc.

But for this specific variable name change, I think a simple variable name fix (just `K_MASK` to `_K_MASK`) will do. Thanks! :)

**Comment by jenriver** [comment]
Resolved by @gear in https://github.com/jax-ml/bonsai/pull/14. Thanks! :)
